rewards/df-challengedf.md

```diff
@@ -0,0 +1,47 @@
+---
+description: >-
+  Weekly prediction challenges for data scientists to compete, launch algorithms, and start earning. 🏆
+---
+
+# Win Competitions in Challenge DF
+
+<figure><img src="../.gitbook/assets/rewards/gif/eat-data.gif" alt=""><figcaption><p>Let's consume some data.</p></figcaption></figure>
+
+## Publish prediction models and earn rewards
+
+If you're eager to consume data and create valuable assets for publishing but unsure of where to begin, fear not! We've got you covered.
+
+Data Farming offers rewards to data scientists who participate in weekly challenges. We provide the rules and rewards, allowing you to focus on delivering high-quality results.  
+
+**Prizes:**
+1st Place - 2500 OCEAN
+2nd Place - 1500 OCEAN
+3rd Place - 1000 OCEAN
+_Total - 5000 OCEAN_
+
+### What are the base rules?
+
+- A fresh challenge begins and concludes every week.
+- At the beginning of each week, a predetermined amount of OCEAN tokens is allocated as rewards.
+- The rewards are divided into specific amounts for the top three positions.
+- The individuals who submit the top three entries will receive rewards, and their names will be showcased within the Data Farming dapp.
```

Comment:
What do they need to submit? The prediction or the algorithm? I think this is not clear in the df documentation we have added in the docs. I see on the GitHub that we're keeping this pace of receiving predictions

Reply:
They need to submit the predictions

Reply:
Addressed here => https://github.com/oceanprotocol/predict-eth/pull/142
Will link to the `predict-eth/challenge-df.md` doc

Reply:
I have updated the doc inside predict-eth, and `df-challengedf.md` to improve knowledge around this.

Reply:
Feedback has been further resolved in `predict-eth` repo.
Thank you everyone.